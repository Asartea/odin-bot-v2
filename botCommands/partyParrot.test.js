const partyParrot = require('./partyParrot')
const botEngine = require('./mockBotEngine.test')

describe('', ()=>{
    test('',()=> {

    })
})
