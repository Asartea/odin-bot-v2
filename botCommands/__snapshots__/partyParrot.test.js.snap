// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`partyparrot snapshot messages including an exclamation point always return the first index in the array 1`] = `"https://cultofthepartyparrot.com/parrots/hd/dadparrot.gif"`;

exports[`partyparrot snapshot messages including an exclamation point always return the first index in the array 2`] = `"https://cultofthepartyparrot.com/parrots/hd/dadparrot.gif"`;

exports[`partyparrot snapshot messages including an exclamation point always return the first index in the array 3`] = `"https://cultofthepartyparrot.com/parrots/hd/dadparrot.gif"`;

exports[`partyparrot snapshot messages including an exclamation point always return the first index in the array 4`] = `"https://cultofthepartyparrot.com/parrots/hd/dadparrot.gif"`;

exports[`partyparrot snapshot messages including an exclamation point always return the first index in the array 5`] = `"https://cultofthepartyparrot.com/parrots/hd/dadparrot.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 1`] = `"https://cultofthepartyparrot.com/parrots/hd/dadparrot.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 2`] = `"http://cultofthepartyparrot.com/parrots/parrot.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 3`] = `"http://cultofthepartyparrot.com/parrots/fiestaparrot.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 4`] = `"http://cultofthepartyparrot.com/parrots/explodyparrot.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 5`] = `"http://cultofthepartyparrot.com/parrots/slomoparrot.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 6`] = `"http://cultofthepartyparrot.com/parrots/hd/dealwithitparrot.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 7`] = `"http://cultofthepartyparrot.com/parrots/tripletsparrot.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 8`] = `"https://i.imgur.com/Q1K4Wzx.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 9`] = `"https://i.imgur.com/rcNgsYF.gif"`;

exports[`partyparrot snapshot when randomInt is mocked, the output remains the same 10`] = `"https://i.imgur.com/RWZuxIa.gif"`;
